tests/MassTransit.Tests/ProcessingTimeout_Specs.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace MassTransit.Tests
+{
+    using System;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+    using Shouldly;
+    using TestFramework;
+    using TestFramework.Messages;
+
+
+    [TestFixture]
+    public class ProcessingTimeout_Specs :
+        InMemoryTestFixture
+    {
+        [Test]
+        public async Task Should_throw_on_timeout()
+        {
+            Task<ConsumeContext<Fault<PingMessage>>> faulted = await ConnectPublishHandler<Fault<PingMessage>>();
+
+            await InputQueueSendEndpoint.Send(new PingMessage());
+
+            await Task.WhenAny(_succeeded, faulted);
+
+            _firstCalled.ShouldBeTrue();

```

Comment:
I'm deprecating use of Shouldly, can you change these to Assert.IsTrue(), etc.

Reply:
Fixed.